exp/addons/api/.import-restrictions

```diff
@@ -0,0 +1,5 @@
+rules:
```

Comment:
Let's additionally add import-restrictions files into all v1beta1 packages where we block everything, i.e.:
```
rules:
  - selectorRegexp: sigs[.]k8s[.]io/controller-runtime
    allowedPrefixes: []
    forbiddenPrefixes:
      - "sigs.k8s.io/controller-runtime"
```